stubs/Pygments/pygments/__init__.pyi

```diff
@@ -1,5 +1,16 @@
-from typing import Any
+from _typeshed import SupportsWrite
+from typing import Any, TypeVar, overload
+
+from .formatters import Formatter
```

Comment:
I don't know how to fix the remaining error...

Reply:
Me neither, stubtest is doing something weird. I might try to debug this more later.

Reply:
Shouldn't this be `.formatter`? (That is, I think this currently relies on implicit reexports formatters)

Not sure why the mypy test doesn't pick this up, maybe we turn off that setting

Reply:
Lol, thanks

```suggestion
from .formatter import Formatter
```